cmd/mount.go

```diff
@@ -391,32 +392,56 @@ var mountCmd = &cobra.Command{
 
 		log.Info("mount: Mounting blobfuse2 on %s", options.MountPath)
 		if !options.Foreground {
+			// redirect libfuse stderr to a temp file
+			f, err := ioutil.TempFile("", "libfuse_stderr_")
```

Comment:
There are customers who mount multiple blobfuse instances on single vm, this means we are creating those many temp files. And in any case if this creation fails we are failing the mount which breaks the current user expectatoins. In cases where it was working fine now there is a probability of that failing.

Reply:
We can sure clean it when child exit. However, I think lots of temp file won't be a problem.

Reply:
It's rarely that create a temp file would fail. And I think retry can solve the problem?

Reply:
agree that probability is low, but failing the mount just because some book-keeping stuff dint go well is not a good idea here.

Reply:
I've made it won't exit when failing to create temp file. And will remove tmp file once parent exit.

Reply:
I just found out how to use PIPE for communication between child and parent, no more temp file.